workers/analyzer/src/main/kotlin/analyzer/AnalyzerWorker.kt

```diff
@@ -61,19 +61,17 @@ internal class AnalyzerWorker(
         val context = contextFactory.createContext(job.ortRunId)
         val envConfigFromJob = job.configuration.environmentConfig
 
-        val (repositoryService, resolvedEnvConfigFromJob) = if (envConfigFromJob != null) {
-            logger.info("Setting up environment from configuration provided in the Analyzer job.")
-            null to environmentService.setUpEnvironment(context, envConfigFromJob)
-        } else {
-            environmentService.findInfrastructureServiceForRepository(context)?.also { serviceForRepo ->
-                logger.info(
-                    "Generating a .netrc file with credentials from infrastructure service '{}' to download the " +
-                            "repository.",
-                    serviceForRepo
-                )
-
-                environmentService.generateNetRcFile(context, listOf(serviceForRepo))
-            } to null
+        val repositoryService = envConfigFromJob?.let {
```

Comment:
Commit message: The most important part of this change is that the environment will now be set up after downloading the repository, please make that clear in the commit message.